pyspecdata/core.py

```diff
@@ -6887,132 +6900,55 @@ def __init__(self,*args,**kwargs):
         self.active_indices = None
         #}}}
         return
-    def parameter_derivatives(self,xvals,set = None,set_to = None):
-        r'return a matrix containing derivatives of the parameters, can set dict set, or keys set, vals set_to'
```

Comment:
don't we still want a `parameter_derivatives` function?  Even if a placeholder that returns an error for now?

Reply:
Yes not sure how this got deleted -- looking back at my notes, at one point this was giving me an error, but I do not think I resolved it by deleting it?

Reply:
I don't see a parameter derivatives function right now